models/classes/media/ItemMediaResolver.php

```diff
@@ -41,7 +41,7 @@ public function __construct($item, $lang)
     public function resolve($url)
     {
         $urlParts = parse_url($url);
-        if (!isset($urlParts['scheme']) || !isset($urlParts['host'])) {
+        if (!isset($urlParts['scheme']) || (!isset($urlParts['host']) && $urlParts['scheme'] !== 'data')) {
```

Comment:
Year field in Copyright block should be updated too

Reply:
Right, I've updated it 👍🏻